doc/release/1.10.2-notes.rst

```diff
@@ -0,0 +1,62 @@
+NumPy 1.10.2 Release Notes
+**************************
+
+This release deals with a number of bugs that turned up in 1.10.1 and
+adds various build and release improvements.
+
+Numpy 1.10.1 supports Python 2.6 - 2.7 and 3.2 - 3.5.
+
+Changes
+=======
+
+Testing with development Numpy versions will raise exceptions for
+``RuntimeWarning`` and ``DeprecationWarning``, released Numpy versions will
+leave them as warnings. Previously, development branches other than
+``master`` would not raise exceptions for these warnings.
```

Comment:
The other change is that 1.10.1 had a bug in exactly this functionality. I was expecting that to be described here - it's more important because it caused test issues for projects depending on numpy.